matrix-core/matrix.tdl

```diff
@@ -1832,15 +1853,15 @@ basic-head-subj-nmc-phrase := head-valence-phrase & basic-binary-headed-phrase &
        [ LOCAL.CAT.VAL [ SUBJ < #synsem >,
                          COMPS < [ LOCAL #slash ] >,
                          SPR #spr ],
-         NON-LOCAL.SLASH 1-dlist & [ LIST < #slash > ] ],
+         NON-LOCAL.SLASH append-list & [ LIST < #slash > ] ],
     NON-HEAD-DTR.SYNSEM #synsem & canonical-synsem &
        [ LOCAL [ CAT [ VAL [ SUBJ olist,
                              COMPS olist,
                              SPR olist ] ] ],
-         NON-LOCAL [ SLASH 0-dlist & [ LIST < > ],
-                     REL 0-dlist,
-                     QUE 0-dlist ] ],
-    C-CONT [ RELS <! !>, HCONS <! !>, ICONS <! !> ] ].
+         NON-LOCAL [ SLASH 0-alist,
```

Comment:
No, it was a 0-dlist.

Reply:
Sorry, I was wrong bringing up `1-dlist`. The main issue I wanted to address was the original constraint was `SLASH 0-dlist & [ LIST < > ]` and you've replaced it with `0-alist`. But, I take it from your other comment that `0-alist` and `LIST < >` are equivalent, so I understand now 🙂

Reply:
Yeah, I don't think that 0-dlist & LIST < > was needed, either! 0-dlist implies LIST < >, I am pretty sure.

Reply:
No, 0-dlist implies `[ LIST 0-1-list & #last, LAST #last ]`.  It can't have `[ LIST < > ]` or else it couldn't be appended to.  Writing `0-dlist & [ LIST < > ]` means: not only is this diff-list empty, but anything appended to it must also be empty.  I think it's unlikely that this is what was intended (and even if it *was* intended, this may lead to grammars that are difficult to maintain).